py_ballisticcalc/trajectory_calc.py

```diff
@@ -356,59 +355,71 @@ def calculate_stability_coefficient(ammo: Ammo, rifle: Weapon, atmo: Atmo):
     return sd * fv * ftp
 
 
-def wind_to_vector(shot: Shot, wind: Wind):
-    sight_cosine = math.cos(shot.zero_angle >> Angular.Radian)
-    sight_sine = math.sin(shot.zero_angle >> Angular.Radian)
-    cant_cosine = math.cos(shot.cant_angle >> Angular.Radian)
-    cant_sine = math.sin(shot.cant_angle >> Angular.Radian)
-    range_velocity = (wind.velocity >> Velocity.FPS) * math.cos(
-        wind.direction_from >> Angular.Radian)
-    cross_component = (wind.velocity >> Velocity.FPS) * math.sin(
-        wind.direction_from >> Angular.Radian)
-    range_factor = -range_velocity * sight_sine
-    return Vector(range_velocity * sight_cosine,
-                  range_factor * cant_cosine + cross_component * cant_sine,
-                  cross_component * cant_cosine - range_factor * cant_sine)
+def wind_to_vector(wind: Wind) -> Vector:
+    """Calculate wind vector to add to projectile velocity vector each iteration:
+        Aerodynamic drag is function of velocity relative to the air stream.
+
+    Wind angle of zero is blowing from behind shooter
+    Wind angle of 90-degree is blowing towards shooter's right
+
+    NB: Presently we can only define Wind in the x-z plane, not any vertical component.
+    """
+    # Downrange (x-axis) wind velocity component:
+    range_component = (wind.velocity >> Velocity.FPS) * math.cos(wind.direction_from >> Angular.Radian)
+    # Cross (z-axis) wind velocity component:
+    cross_component = (wind.velocity >> Velocity.FPS) * math.sin(wind.direction_from >> Angular.Radian)
+    return Vector(range_component, 0, cross_component)
 
 
 def create_trajectory_row(time: float, range_vector: Vector, velocity_vector: Vector,
-                          velocity: float, mach: float, windage: float, weight: float, flag: int):
+                          velocity: float, mach: float, windage: float, look_angle: float,
+                          density_factor: float, drag: float, weight: float, flag: int):
     drop_adjustment = get_correction(range_vector.x, range_vector.y)
     windage_adjustment = get_correction(range_vector.x, windage)
     trajectory_angle = math.atan(velocity_vector.y / velocity_vector.x)
 
-
     return TrajectoryData(
```

Comment:
does it resturn an adjustments relative to zero line?

Reply:
As of current version: drop_adjustment is relative to line of sight, so that it will be valid regardless of look-angle.